ReSharper.FSharp/test/src/FSharp.Tests/Actions/FSharpExtendSelectionTest.fs

```diff
@@ -34,3 +34,8 @@ type FSharpExtendSelectionTest() =
     [<Test>] member x.``Let - Top level 01``() = x.DoNamedTest()
     [<Test>] member x.``Let - Top level 02 - Rec, other``() = x.DoNamedTest()
     [<Test>] member x.``Let - Top level 03 - Rec, attrs``() = x.DoNamedTest()
+    
+    [<Test>] member x.``Infix expr 01``() = x.DoNamedTest()
+    [<Test>] member x.``Infix expr 02``() = x.DoNamedTest()
+    [<Test>] member x.``Infix expr 03``() = x.DoNamedTest()
+    [<Test>] member x.``Infix expr 04``() = x.DoNamedTest()
```

Comment:
Could you add some description to the test names please? Something like `Nested` for the last two, and `Function`, `Argument` for all.